airflow/models.py

```diff
@@ -2483,6 +2483,8 @@ class DAG(BaseDag, LoggingMixin):
 
     :param dag_id: The id of the DAG
     :type dag_id: string
+    :param description: The description for the DAG to e.g. be shown on the webserver
```

Comment:
[optiona] could be nice to support markdown here, allowing for styling, links, images, ...